modules/ROOT/pages/dw-core-functions-match.adoc

```diff
@@ -8,7 +8,7 @@ endif::[]
 [[match1]]
 == match&#40;String, Regex&#41;: Array<String&#62;
 
-Uses a regular expression (regex) to match a string and then separates it into
+Uses a Java regular expression (regex) to match a string and then separates it into
```

Comment:
Looking at this again, the sentence seems awkward. I suggest making "string" singular.

Uses a Java regular expression (regex) to match a string and then separate it into...

Reply:
"string" is already singular here. 

The unnamed subject is `match`, which uses ... and then separates... , so leaving as is. The regex is not doing the separating, the function is.